sunpy/coordinates/tests/test_ephemeris.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+
+import astropy.units as u
+from astropy.tests.helper import assert_quantity_allclose
+from astropy.coordinates import EarthLocation
+
+from ..ephemeris import *
```

Comment:
Makes sense to me.  However, I'll note that I actually copied that pattern from an existing test file in SunPy, so we're not consistent throughout.